std_msgs/Header header

string[] int32_paths
int32[] int32_values

string[] float64_paths
float64[] float64_values

string[] string_paths
string[] string_values